Magento_Theme/templates/footer/footer-links.phtml

```diff
@@ -0,0 +1,191 @@
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('Payments') ?>
+                        </a>
+                    </li>
+                    <li class="list__item ">
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('Delivery cost') ?>
+                        </a>
+                    </li>
+                    <li class="list__item ">
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('Returns and exchanges') ?>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </li>
+        <li class="
+                dropdown-list__item
+                dropdown-list__item--collapse
+            "
+        >
+            <button class="dropdown-list__label "
+                    aria-expanded="false"
+                    aria-controls="dropdown-worth"
+                    data-dropdown="dropdown-worth"
+            >
+                <?= /* @escapeNotVerified */ __('Why is worth it') ?>
+                <svg class="dropdown-list__icon">
+                    <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#angle-down') ?>"></use>
+                </svg>
+            </button>
+            <div id="dropdown-worth" class="dropdown-list__content " data-content="dropdown-worth" aria-hidden="true">
+                <ul class="list footer__links-list">
+                    <li class="list__item ">
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('30 days for return') ?>
+                        </a>
+                    </li>
+                    <li class="list__item ">
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('Guarantee') ?>
+                        </a>
+                    </li>
+                    <li class="list__item ">
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('454 Brands') ?>
+                        </a>
+                    </li>
+                    <li class="list__item ">
+                        <a href="#" class="link link--secondary">
+                            <?= /* @escapeNotVerified */ __('Discounts') ?>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </li>
+        <li class="
+                dropdown-list__item
+                dropdown-list__item--collapse
+            "
+        >
+            <button class="dropdown-list__label "
+                    aria-expanded="false"
+                    aria-controls="dropdown-alpaca"
+                    data-dropdown="dropdown-alpaca"
+            >
```

Comment:
missing button type

Reply:
Did you add type? I can't see. I think should be better to add.

Reply:
You're right, now added!